extensions/standard-processors/tests/unit/ProcessorTests.cpp

```diff
@@ -207,6 +209,60 @@ TEST_CASE("Test GetFile Ignore", "[getfileCreate3]") {
   prev++;
 }
 
+TEST_CASE("TestConnectionFull", "[ConnectionFull]") {
+  TestController testController;
+  LogTestController::getInstance().setDebug<minifi::processors::GenerateFlowFile>();
+  std::shared_ptr<core::ContentRepository> content_repo = std::make_shared<core::repository::VolatileContentRepository>();
+  std::shared_ptr<core::Processor> processor = std::make_shared<org::apache::nifi::minifi::processors::GenerateFlowFile>("GFF");
```

Comment:
Use `auto` to reduce type name clutter when it's obvious from the context, like in the case of `make_shared`, `make_unique` and iterators.

Reply:
As you can see both shared ptrs are base class type, so using auto wouldn't be equivalent.

Reply:
True, nevermind then